src/components/About.js

```diff
@@ -48,11 +68,15 @@ export default function About(props) {
         </div>
 
         <div className={imgStyles.imgWrapper}>
-          <div
-            role="img"
-            aria-label="blonde woman in black and white outfit"
-            className={imgStyles.img}
-          ></div>
+          {showImage ? (
+            <div
+              role="img"
+              aria-label="blonde woman in black and white outfit"
+              className={imgStyles.img}
+            ></div>
+          ) : (
+            <div className={placeholderStyle} ref={placeholder}></div>
```

Comment:
The placeholder is an empty div, it could have a background or something, while the image is being loaded

Reply:
Yeah I didn't get to styles yet, you might have noticed a big yellow square I set as a placeholder 😄